spec/features/super_admin/view_organisation_page_spec.rb

```diff
@@ -43,20 +48,17 @@
 
     it 'shows the number of locations' do
       within('#location-count') do
-        expect(page).to have_content('1')
+        expect(page).to have_content('3')
```

Comment:
just commenting on capybara's design here...
does it really modify `page` when entering a `within` block? I'd have expected it to pass through a fragment to the block.

:roll_eyes: